jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -108,29 +110,15 @@ async def put(self):
             )
 
 
-class CommitHandler(BaseHandler):
-    @tornado.web.authenticated
-    async def post(self):
-        try:
-            body = self.get_json_body()
-            response = await self.mount_client.commit(body)
-            get_logger().debug(f"Commit: {response}")
-            self.finish(response)
-        except Exception as e:
-            get_logger().error(f"Error committing {body}.", exc_info=True)
-            raise tornado.web.HTTPError(
-                status_code=getattr(e, "code", 500),
-                reason=f"Error committing {body}: {e}.",
-            )
-
-
+# probably not needed anymore
```

Comment:
Mount server had the functionality to commit dirty files in the mount, but we never exposed this to the user in the extension. The future of this capability is probably a product question.